.github/workflows/release.yml

```diff
@@ -64,6 +65,9 @@ jobs:
         registry: quay.io
     - name: Prepare release
       run: |
+        git config --global user.name "Mykhailo Kuznietsov"
+        git config --global user.email "mkuznets@redhat.com"
+        export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
```

Comment:
I suppose we also need to add `AKAMAI_CHE_AUTH` to the env: section below to expose it from secret -> env var?